### PR TITLE
feat(DataGraph): Add dynamic mapping

### PR DESCRIPTION
**Pull request by GordonSmith**
Add better support for graph wasm loading

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review

File: packages/graph/src/graph2/dataGraph.ts [hunk 1]
```diff
@@ -87,6 +88,13 @@ export class DataGraph extends Graph2 {
 
     constructor() {
         super();
+        this
+            .vertexRenderer(Vertex3)
+            .centroidRenderer(CentroidVertex3)
+            .edgeColor("#287EC4" as any)
```
Comment (jbrundage) [inline packages/graph/src/graph2/dataGraph.ts:94]: Why `as any`?

Comment (GordonSmith) [reply packages/graph/src/graph2/dataGraph.ts:94]: For a while it had been PP as a number
I will fix.

File: packages/util/src/platform.ts [hunk 2]
```diff
@@ -6,3 +6,18 @@ export const root: any = new Function("return this;")(); //  Prevent bundlers fr
...
+    for (let i = scripts.length - 1; i >= 0; --i) {
+        const script: any = scripts[i];
+        const attrVal: string = script.getAttribute.length !== undefined ? script[attr] : script.getAttribute(attr) || "";
+        const idx = attrVal.indexOf(partial);
+        if (idx >= 0) {
+            return attrVal.substring(0, idx);
+        }
+    }
+    return "";
+}
```
Comment (jbrundage) [inline packages/util/src/platform.ts:23]: This function could be:
```js
export function getScriptSrc(partial: string) {
    for(const script of document.scripts) {
        if(!script.src)continue;
        const idx = script.src.indexOf(partial);
        if(idx !== -1)return script.src.substring(0, idx);
    }
    return "";
}
```

Comment (GordonSmith) [reply packages/util/src/platform.ts:23]: Yes (need to iterate backwards though)

**Pull request merged**